nx-X11/programs/Xserver/hw/nxagent/NXdispatch.c

```diff
@@ -199,12 +199,9 @@ extern int nxagentFindClientResource(int, RESTYPE, void *);
 
 
 void
-InitSelections()
+InitSelections(void)
 {
-    if (CurrentSelections)
-	free(CurrentSelections);
-    CurrentSelections = (Selection *)NULL;
-    NumCurrentSelections = 0;
+    xorg_InitSelections();
```

Comment:
Ahhhh... now I get what you are doing.

Ok. These xorg_* namespacing commits, they will work copyright wise...